# Comparative computational RNA analysis of cardiac-derived progenitor cells and their extracellular vesicles
## TotalRNAseq analysis  (miRNA, lncRNA, mRNA) of CPCs and their EVs. Genomics 2022.
### [Manuscript](https://www.sciencedirect.com/science/article/pii/S0888754322000945?via%3Dihub)
### Deposited data: 
#### miRNA: [GSE202347](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202347)
#### totalRNA: [GSE202345](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202345)

### Abstract:
Stem/progenitor cells, including cardiac-derived c-kit+ progenitor cells (CPCs), are under clinical evaluation for treatment of cardiac disease. Therapeutic efficacy of cardiac cell therapy can be attributed to paracrine signaling and the release of extracellular vesicles (EVs) carrying diverse cargo molecules. Despite some successes and demonstrated safety, large variation in cell populations and preclinical/clinical outcomes remains a problem. Here, we investigated this variability by sequencing coding and non-coding RNAs of CPCs and CPC-EVs from 30 congenital heart disease patients and used machine learning methods to determine potential mechanistic insights. CPCs retained RNAs related to extracellular matrix organization and exported RNAs related to various signaling pathways to CPC-EVs. CPC-EVs are enriched in miRNA clusters related to cell proliferation and angiogenesis. With network analyses, we identified differences in non-coding RNAs which give insight into age-dependent functionality of CPCs. By taking a quantitative computational approach, we aimed to uncover sources of CPC cell therapy variability.

Code by JR Hoffman
